settings/src/components/webauthn/register-key.js

```diff
@@ -79,16 +79,18 @@ export default function RegisterKey( { onSuccess, onCancel } ) {
 
 	return (
 		<form onSubmit={ onRegister }>
+			<p className="wporg-2fa__screen-intro">Give the security key a name.</p>
+
 			<TextControl
-				label="Give the security key a name"
+				label="Name"
 				onChange={ ( name ) => setKeyName( name ) }
 				value={ keyName }
 				required
 			/>
 
 			<div className="wporg-2fa__submit-actions">
 				<Button type="submit" variant="primary" disabled={ registerCeremonyActive }>
-					Register Key
+					Register
 				</Button>
 
 				<Button variant="secondary" onClick={ onCancel }>
```

Comment:
The designs have a borderless button:

| Designs | This PR |
| --- | --- |
| <img width="184" alt="Screenshot 2023-08-11 at 3 30 09 PM" src="https://github.com/WordPress/wporg-two-factor/assets/1657336/5982d8a6-3b45-459c-9725-029ce31b3dda"> | <img width="340" alt="Screenshot 2023-08-11 at 3 32 07 PM" src="https://github.com/WordPress/wporg-two-factor/assets/1657336/ad42ed7b-747a-433b-a517-138e933ab2fa"> |

I think that would make the button spacing feel better. If we can't remove the border for whatever reason, i would suggest making the space between them smaller. It feels kind of randomly space.


 |

Reply:
I'll take another look. There was [feedback suggesting a tertiary button might be appropriate](https://github.com/WordPress/wporg-two-factor/issues/194#issuecomment-1669120695). I feel like using the theme styles is the right thing to do though.

Reply:
Removed

Reply:
Tertiary buttons do match the borderless style. I've replaced all the appropriate usages.